interface-definitions/ipsec-settings.xml

```diff
@@ -9,12 +9,54 @@
               <help>Global IPsec settings</help>
             </properties>
             <children>
+              <leafNode name="enable-unencrypted-mainmode-messages">
```

Comment:
Can we probably rename this leafNode to: `unencrypted-mainmode`? CLI should be not to long. B/c everything else is stated in the help message.

Reply:
good idea